site/layouts/partials/anchor.html

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Where does it come from? Maybe include the info in the PR description for future reference.

Reply:
Refer to the Clickable headers svg of kubernetes docs and implement it through Heading render hooks in hugo.

Reply:
Thanks for clarifying